### ISSUE TITLE
Generated ONNX not compatible with opset 9

### ISSUE THREAD

**Issue by mnboos** [onnx/keras-onnx#57]
The default seems to be opset 9. Unfortunately, the generated ONNX contains for example the attribute `spatial` in a `BatchNormalization` node, which was valid for opset 7 (https://github.com/onnx/onnx/blob/master/docs/Changelog.md#BatchNormalization-7) but not for the current opset 9 (https://github.com/onnx/onnx/blob/master/docs/Operators.md#BatchNormalization).

Microsofts ONNX runtime does not even load such a model, which is correct in my opinion.

**Comment by jiafatom** [comment]
@mnboos As of today, please install keras2onnx directly from github (which supports opset 9), not from pypi. We will release pypi package soon.

**Comment by mnboos** [comment]
Okay, thanks for the clarification.
